lib/apivore/rspec_builder.rb

```diff
@@ -32,7 +32,7 @@ def apivore_build_path(path, data)
           raise URI::InvalidURIError, "No substitution data found for {#{key}} to test the path #{path}.\nAdd it via an:\n  apivore_setup '<path>', '<method>', '<response>' do\n    { '#{key}' => <value> }\n  end\nblock in your specs.", caller
         end
       end
-      path
+      path + (data['_query_string'] ? "?#{data['_query_string']}" : '')
```

Comment:
@hornc I had tried making it conditional and it did not solve the problem. I didn't spend too much time on fixing  the test yet (the code runs correctly) but did not immediately how it should be failing right here. Rubygems site is broken atm to re-test it.